addons/mail/static/src/messaging/component/ellipsis/ellipsis.js

```diff
@@ -0,0 +1,32 @@
+odoo.define('mail.messaging.component.Ellipsis', function (require) {
+'use strict';
+
+const { Component } = owl;
+
+class Ellipsis extends Component {}
```

Comment:
This should actually not be called Ellipsis, but have the notion of `TypingIndicator` inside the name.

Reply:
Reasoning is that I would except a component named ellipsis to take a text as parameter and add ellipsis if the text is overflowing.